compute/standard-deviation/docs/modules/ROOT/pages/compute-standard-deviation.adoc

```diff
@@ -269,6 +269,14 @@ The file is uploaded and is ready to be used:
 
 Now that the cluster-side modules are ready to use, you can run the client to trigger the process of calculating the standard deviation of customer satisfaction scores.
 
+. Open the link:https://cloud.hazelcast.com/[{hazelcast-cloud} console].
+
+. Go to *Connect Client* > *Advanced setup*, and click *Download keystore file*.
+
+. Extract the files and copy the `client.keystore` and `client.truststore` files to the `client/src/main/resources` directory.
```

Comment:
Minor point. I had to create the resources directory. It didn't previously exist.

Reply:
Thanks - I added a file to the directory so that we can commit it.